doc/readme.md

```diff
@@ -0,0 +1,335 @@
+        levelAPI.loadLevel(LEVELSIZE);
+        createSystems();
+    }
+```
+
+Wenn Sie nun das Spiel starten, sehen Sie zwar das Level im Dungeon, aber Ihren Helden noch nicht.
+
+### Held zeichnen
+
+Damit der Held im Spiel auch angezeigt werden kann, braucht er ein `AnimationComponent`, denn dann wird das `DrawSystem` den Helden an seiner aktuellen Position zeichnen. Für das `AnimationComponent` benötigen wir zwei `Animation`s. Eine für `idleRight` und eine für `idleLeft`. `Animation`s sind eine Reihe an Bildern, die abwechselnd angezeigt werden. 
+
+Animation können wir mit dem `AnimationBuilder` erzeugen, indem wir `AnimationBuilder.buildAnimation` aufrufen und als Parameter das Verzeichnis mit den Bildern übergeben, die zu einer Animation zusammengefasst werden sollen. Das `AnimationComponent` wird im Konstruktor erstellt.
+
+``` java
+package ecs.entities;
+
+import dslToGame.AnimationBuilder;
+import ecs.components.AnimationComponent;
+import ecs.components.PositionComponent;
+import graphic.Animation;
+import tools.Point;
+
+public class MyHero extends Entity {
+
+    public MyHero() {
+        super();
+        new PositionComponent(this, new Point(0, 0));
+        Animation idleLeft = AnimationBuilder.buildAnimation("character/knight/idleLeft");
+        Animation idleRight = AnimationBuilder.buildAnimation("character/knight/idleRight");
+        new AnimationComponent(this, idleLeft, idleRight);
+    }
+}
+
+```
+Wenn Sie das Spiel jetzt starten, sollten SIe ihren Helden sehen:
+
+![Animation](figs/animation.gif)
+
+### Held bewegen
+
+Damit der Held vom `VelocitySystem` bewegt werden kann, benötigt er das `VelocityComponent`. Das `VelocityComponent` speichert die Maximalgeschwindigkeit des Helden, sowie die aktuelle geschwindigkeit des Helden.
+
+Damit wir den Helden auch per Tastatur steuern können, benötigt er das `PlayableComponent`, denn dann wird das `PlayerSystem` die Werte des Helden auf Knopfdruck aktualisieren.
+
+*Hinweis: Mehr zur Konfiguration der Tastenbelegung erfahren Sie [hier](configuration/readme.md).*
+
+Genau wie die anderen `Component`s fügen wir das `VelocityComponent` und das `PlaybaleComponent` zum Konstruktor hinzu.
+
+``` java
+package ecs.entities;
+
+import dslToGame.AnimationBuilder;
+import ecs.components.AnimationComponent;
+import ecs.components.PlayableComponent;
+import ecs.components.PositionComponent;
+import ecs.components.VelocityComponent;
+import graphic.Animation;
+import tools.Point;
+
+public class MyHero extends Entity {
+
+    public MyHero() {
+        super();
+        new PositionComponent(this, new Point(0, 0));
+        Animation idleLeft = AnimationBuilder.buildAnimation("character/knight/idleLeft");
+        Animation idleRight = AnimationBuilder.buildAnimation("character/knight/idleRight");
+        new AnimationComponent(this, idleLeft, idleRight);
+        new PlayableComponent(this);
+        float xSpeed = 0.3f;
```

Comment:
@AMatutat Stilfrage. Macht es Sinn, für die beiden Floats extra Variablen zu verwenden? Die werden nirgendwo sonst nochmal verwendet, und sind obendrein Magic Numbers. Dann können sie eigentlich auch gleich direkt in Z. 244 (VelociRaptor) auftauchen?

Reply:
Wenn das "echter" Code wäre, würde ich das machen. Für einen Quickstart finde ich das so netter als es direkt in den Konstrukten zu kippen. Ist etwas weniger magisch. 
Vielleicht die Namen anpassen is `movementspeedOnX` oderso